Makefile

```diff
@@ -16,8 +16,8 @@ install-test-requirements:
 exec-test:
 	cat pytest.ini
 	@# use it like this: `make exec-test TEST_TARGET="tests/unit/"`
-	PYTHONPATH=$(CURDIR) pytest-2 $(TEST_TARGET)
-	PYTHONPATH=$(CURDIR) pytest-3 $(TEST_TARGET)
+	PYTHONPATH=$(CURDIR) pytest-2 $(TEST_TARGET) --fulltrace
+	PYTHONPATH=$(CURDIR) pytest-3 $(TEST_TARGET) --fulltrace
```

Comment:
Nice, I haven't known about it, but it seems to actually be `--full-trace`

Reply:
The reason I did this, is that ^c didn't yield any traceback. Will update.